### DISCUSSION TITLE
Wo finde ich Erklärungen bezüglich Firmware Änderungen?

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#555] [Q&A] [answered]
Bei den **Features** gibt es (verständliche) Erläuterungen wenn ich auf dem Link klicke.
Kann ich davon ausgehen, dass unter **Under the hood** handelt es sich um Themen die mich als Durchschnittsanwender nicht zu interessieren haben?
Denn manchmal, ist der Titel verführerisch... 🙈 
![Capture](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/33ac697f-32be-461d-8dc0-9dfa556626dd)

**Comment by schlimmchen** [comment, marked as answer]
In den Releases werden commits, die mit "Feature:" oder "Fix:" beginnen, entsprechend automatisch einsortiert. Alles andere ist "under the hood". Ich hab begonnen meine PR so zu gestalten, dass helgeerbe beim Mergen und Releasen sehr wahrscheinlich dies so tut, dass meine PRs entsprechend in den Releases einsortiert werden. Das heißt aber nicht, dass es auch unter "under the hood" interessante Themen geben kann. Die sind dann ggf. nur nicht mit "Feature:" oder "Fix:" in der commit-Nachricht versehen worden vom Autor oder von helgeerbe.

Dass du "add webapp" eingekreist hast deutet darauf hin, dass du diesen commit spannend findest. Dahinter verbirgt sich aber nur eine aktualisierte Version der Web Applikation, die du schon kennst. Da die WebApp nicht während des Firmware-Builds gebaut wird (ein Mangel meiner Meinung nach...), muss sie vorher manuell gebaut werden, sodass dass Transpilat dann in die Firmware einkompiliert wird. Tut man das, hat man drei geänderte Dateien, die zur WebApp gehören. Die fügt man dann mit einem "add webapp" commit dem Repo hinzu, sodass insb. Github die aktuelle Version der WebApp mit einkompiliert.

**Reply by Manos1966** [reply]
Danke @schlimmchen  fuer die Erklaerungen.
Kein Wunder also, dass ich nichts verstanden habe beim Versuch mehr ueber das "add webapp" zu finden 🙈 

Frohe Festtage!  🎄
